src/should-forward-property.js

```diff
@@ -600,17 +26,18 @@ const isCustomAttribute = RegExp.prototype.test.bind(
 )
 
 const hasItem = (list, name) => list.indexOf(name) !== -1
-const isHtmlProp = name => hasItem(htmlProps, name)
+const isHtmlProp = (name, tagName) => {
+  const elementAttributes = reactHTMLAttributes[tagName]
+
+  return hasItem([...globalReactHtmlProps, ...elementAttributes], name)
```

Comment:
Because this code is run a lot, I want to be very considerate of performance here. Could we write this function like this instead:

```javascript
function isHtmlProp(name, tagName) {
  const elementAttributes = reactHTMLAttributes[tagName]
  // elementAttributes could be undefined, so we do defined check here
  return hasItem(globalReactHtmlProps, name) || elementAttributes === undefined ? false : hasItem(elementAttributes, name)
}
```

And actually, while we're here, we could probably speed this up a little bit by removing the `hasItem` helper function and just use `x.indexOf(y) !== -1` everywhere. Am I over-optimizing? Hard when we don't really have benchmarking set up. I'm just freaked out by #43 :wink: and this is definitely "hot code" so I want to make sure it's fast :)